src/_sass/layout.scss

```diff
@@ -477,6 +477,20 @@ h4,
 .blogContent h3 {
   color: $text-color;
 }
+
+.docsHeader {
+  //display: flex;
+  //flex-direction: column-reverse;
+
+  [role="doc-subtitle"] {
+    margin-top: -1.5rem;
+    text-transform: uppercase;
+    font-size: 16px;
+    color: #9a9a9a;
+    //margin-bottom: -0.5rem;
```

Comment:
(nit) if you're not still fiddling with styles here, you can delete the commented out styles

Reply:
My CSS incompetence exposed! 🙈